modules/quickstart-sap/pages/prepare-clients.adoc

```diff
@@ -0,0 +1,84 @@
+[[quickstart-sap-setup]]
+= Preparing Clients
+
+Your SAP cluster requires several client systems.
+Prepare your clients on physical or virtual hardware, and ensure you have {sles}{nbsp}15 for SAP installation media ready on your {productname} Server.
+You will not be able to create an SAP cluster without the {sles} SAP extension, as it provides tooling specific to SAP.
+
+One of the key features of SAP is high availability of the cluster.
+Every component within an SAP cluster has redundancy and failover protection.
+When you are preparing your clients, ensure you have enough hardware and infrastructure to allow for this.
+For more information about hardware requirements, see https://documentation.suse.com/sles-sap/15-SP1/html/SLES4SAP-quick/cha-s4s-plan.html#sec-s4s-hardware[].
+
+For more information about the clients you need to set up for an SAP cluster, see https://documentation.suse.com/sbp/all/html/SBP-SAP-AzureSolutionTemplates/index.html[].
```

Comment:
@Loquacity This link is specific for Azure Templates, but the SAP and Cluster requirements varies depending on the platform. I would point it to the SUSE Best Practices page, where we have many SAP related documents: https://documentation.suse.com/sbp/all/